src/components/DigitalCredentials/CredentialsTable.vue

```diff
@@ -1,8 +1,9 @@
 <template>
-  <div
-    id="credentials-table"
-    class="mt-8"
-  >
+  <div>
+    <h1 class="mb-4">
```

Comment:
The coding style in this UI is mostly to declare top whitespace (instead of bottom whitespace or worse, both!). I'd prefer to see mt-4 on line 7 instead.

Reply:
Thanks for changing this. Looks good.